### PR TITLE
Update docker content part 2

### PR DESCRIPTION
**Pull request by mihaitusa**
change_double_quotes_and_expressions_syntax

### CODE REVIEW

File: content/io/cloudslang/docker/cadvisor/evaluate_resource_usage.sl [hunk 1]
```diff
@@ -9,15 +9,18 @@
...
+#   - memory_usage - calculated memory usage of container; if machine_memory_limit is given use lower of container
+#                    memory limit and machine memory limit to calculate
 #   - cpu_usage - calculated CPU usage of container
 #   - throughput_rx - calculated network Throughput Tx bytes
 #   - throughput_tx - calculated network Throughput Rx bytes
 #   - error_rx - calculated network error Tx
 #   - error_tx - calculated network error Rx
 # Outputs:
-#   - errorMessage - returnResult if returnCode == -1 or statusCode != 200
+#   - errorMessage - returnResult if returnCode == '-1' or statusCode != '200'
```
Comment (gituser173) [inline content/io/cloudslang/docker/cadvisor/evaluate_resource_usage.sl]: shouldn't this use ${}?

File: content/io/cloudslang/docker/swarm/clear_cluster.sl [hunk 2]
```diff
@@ -64,8 +67,7 @@ flow:
     - agent_forwarding:
         required: false
     - docker_options:
-        default: >
-          '-H tcp://' + swarm_manager_ip + ':' + swarm_manager_port
+        default: "${'-H tcp://' + swarm_manager_ip + ':' + swarm_manager_port}"
```
Comment (gituser173) [inline content/io/cloudslang/docker/swarm/clear_cluster.sl]: maybe remove the surrounding quotes, but only if it compiles successfully

File: content/io/cloudslang/docker/swarm/start_manager.sl [hunk 3]
```diff
@@ -65,20 +66,20 @@ flow:
     - run_manager_container:
         do:
           containers.run_container:
-            - container_params: "'-p ' + swarm_port + ':2375'"
-            - container_command: "'manage token://' + cluster_id"
-            - image_name: swarm_image
+            - container_params: "${'-p ' + swarm_port + ':2375'}"
```
Comment (gituser173) [inline content/io/cloudslang/docker/swarm/start_manager.sl]: same here

File: content/io/cloudslang/docker/cadvisor/evaluate_resource_usage.sl (hunk 1 above)
Comment (mihaitusa) [inline content/io/cloudslang/docker/cadvisor/evaluate_resource_usage.sl]: @gituser173  No... in description section we decide not to add ${}

**Comment by oritstone** [comment]
@mihaitusa @gituser173 can it be merged?

**Comment by mihaitusa** [comment]
@oritstone  Yes... it can be merged.

**Comment by oritstone** [comment]
So go ahead and merge :)

**Pull request merged**